cmd/algocfg/profileCommand.go

```diff
@@ -141,7 +133,7 @@ var listProfileCmd = &cobra.Command{
 
 var setProfileCmd = &cobra.Command{
 	Use:   "set",
-	Short: "Create or update config.json files with a specified profile.",
+	Short: "Create config.json file from a profile.",
```

Comment:
and then here, we can say "set" instead of "create"?

Reply:
```suggestion
	Short: "Set config.json file from a profile.",
```